mollie/refunds.go

```diff
@@ -0,0 +1,243 @@
+package mollie
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/google/go-querystring/query"
+)
+
+// Refund describe a refund for a certain payment.
+type Refund struct {
+	Resource         string        `json:"resource,omitempty"`
+	ID               string        `json:"id,omitempty"`
+	Amount           *Amount       `json:"amount,omitempty"`
+	SettlementID     string        `json:"settlementId,omitempty"`
+	SettlementAmount *Amount       `json:"settlementAmount,omitempty"`
+	Description      string        `json:"description,omitempty"`
+	Metadata         interface{}   `json:"metadata,omitempty"`
+	Status           *RefundStatus `json:"status,omitempty"`
+	Lines            *OrderLines   `json:"lines,omitempty"` //TODO: this property should be from order.go
+	PaymentID        string        `json:"paymentId,omitempty"`
+	OrderID          string        `json:"orderId,omitempty"`
+	CreatedAt        *time.Time    `json:"createdAt,omitempty"`
+	Links            *RefundLinks  `json:"_links,omitempty"`
+}
+
+// RefundList describes how a list of refunds will be retrieved by Mollie.
+type RefundList struct {
+	Count    int `json:"count,omitempty"`
+	Embedded struct {
+		Refunds []Refund
```

Comment:
For consistency here please also add the `json:` tag.

Reply:
I missed it. Ok, I'll add it.
But here 
https://github.com/VictorAvelar/mollie-api-go/blob/e2059ee18968d3fa2d35beb6996a120224186c89/mollie/chargebacks.go#L52
https://github.com/VictorAvelar/mollie-api-go/blob/e2059ee18968d3fa2d35beb6996a120224186c89/mollie/methods.go#L48
https://github.com/VictorAvelar/mollie-api-go/blob/e2059ee18968d3fa2d35beb6996a120224186c89/mollie/payments.go#L221
Those are lacking the tag too. Do you want me to add the tag for those lines too?

Reply:
I mean Go will expand the JSON name as the property name so technically it should be fine, just to keep it consistent we can leave this one without a tag. 😄